commitizen/cz/customize/customize.py

```diff
@@ -68,16 +68,16 @@ def message(self, answers: dict) -> str:
         else:
             return message_template.render(**answers)
 
-    def example(self) -> str | None:
-        return self.custom_settings.get("example")
+    def example(self) -> str:
+        return self.custom_settings.get("example") or ""
```

Comment:
You can set the default for the `.get`, right

Reply:
ah, yes, miss this one (also missed the reviewed...). Let me fix it now!

Reply:
mypy doesn't seem to be smart enough to make it right. We probably need to rewrite the setting thing a bit to make it work 🤔 But let's keep it for now